dist/lib/testing/deployed.js

```diff
@@ -31,9 +31,9 @@ var Deployed = {
 
   // Pulled from ethereumjs-util, but I don't want all its dependencies at the moment.
   toChecksumAddress: function (address) {
-    var web3 = new Web3();
+    var appchain = new AppChain();
```

Comment:
See a lot of `var`. Possible to use `const` or `let` instead?

Reply:
Fixed